src/pubsub/ua_pubsub_manager.c

```diff
@@ -773,6 +777,14 @@ UA_PubSubManager_delete(UA_Server *server, UA_PubSubManager *pubSubManager) {
     TAILQ_FOREACH_SAFE(tmpSDS1, &server->pubSubManager.subscribedDataSets, listEntry, tmpSDS2){
         UA_Server_removeStandaloneSubscribedDataSet(server, tmpSDS1->identifier);
     }
+
+#if defined(UA_ENABLE_PUBSUB_ENCRYPTION) && defined(UA_ENABLE_PUBSUB_SKS)
+    UA_PubSubKeyStorage *keyStorage, *keyStorageTemp;
```

Comment:
If you call the variables `ks, ksTmp`, then the foreach fits into a single line.

Reply:
shorten the pointer names